packages/components/package.json

```diff
@@ -50,20 +53,26 @@
 		"@wp-g2/styles": "^0.0.160",
 		"@wp-g2/utils": "^0.0.160",
 		"classnames": "^2.2.5",
+		"create-emotion": "^10.0.27",
 		"dom-scroll-into-view": "^1.2.1",
 		"downshift": "^6.0.15",
+		"emotion-theming": "^10.0.27",
 		"gradient-parser": "^0.1.5",
 		"highlight-words-core": "^1.2.2",
+		"hoist-non-react-statics": "^3.3.2",
 		"lodash": "^4.17.19",
 		"memize": "^1.1.0",
+		"mitt": "^2.1.0",
 		"moment": "^2.22.1",
 		"re-resizable": "^6.4.0",
 		"react-dates": "^17.1.1",
+		"react-merge-refs": "^1.1.0",
```

Comment:
We have our own merge ref hook in the compose package that is better than this one because it handles dependencies better.